src/ruby/ext/symengine/symengine.c

```diff
@@ -1,4 +1,3 @@
-#include "extconf.h"
```

Comment:
It's generated right after executing extconf.rb and contains the outputs of have_* functions as HAVE_* macros. The contents are given at the end of every mkmf.log file. like
``` cpp
extconf.h is:
/* begin */
1: #ifndef EXTCONF_H
2: #define EXTCONF_H
3: #define HAVE_CWRAPPER_H 1
4: #endif
/* end */
```